docs/articles/licensing/gplv3/gplv3-blacklist.rst

```diff
@@ -0,0 +1,86 @@
+GPLv3 licensed production software Vs automotive industry
+---------------------------------------------------------
+
+Modern cars have around 100 million lines of code running on them,12 with 70% 
```

Comment:
SWF might provide some context though @GordanM perhaps. Though I do think we can reduce the verbiage a bit and write something like "The GPLv3's anti-tivoization clause is interpreted as meaning one has to provide installation information to modify source code if you ship GPLv3 licensed software. The installation information requirement means that car makers cannot have an encrypted boot sequence in the vehicle without giving away their private keys. This compromises security and safety in the vehicle and as a consequence, GPLv3 software is often blacklisted by car companies. This documentation describes a workaround to this problem through using older GPLv2 licensed software."

Reply:
@jeremiah thanks a lot! That sounds good, as long as @FlorentRevest comment is addressed as well (what is actually tivoization, a short explanation and link would be enough). 

Always the smooth talker :)

Reply:
I find it super weird to on the one hand to say "GPLv3's tivoization clause compromises security and safety" and as a sollution to that to propose some old ass versions of the software which still are under a GPLv2 but are unmaintained and full of security holes, it just doesn't make any sense to me, it's basically lying.

Reply:
You're absolutely right @rpannek, but don't shoot the messenger. :-) 

You surely understand though that a car maker might worry about their public reputation if there is an accident that was caused by user modified software. That is not unreasonable, some companies, like Volvo, base their brand on safety, of course they want to preserve the brand. Disallowing user modification is one way to help ensure safety. I don't think anyone is saying what you put in quotes however and I think that showing other users how to use yocto's license filtering mechanism is a bad thing as it can be used for other purposes.

I am not really sure about this whole enterprise frankly and I think that is what you're pointing to in some way Richard. Why are we doing this? We are completely foss so why are we telling folks how to do a non-foss configuration in our foss product? Shouldn't this be done on a case-by-case basis internally for customers? There we can warn them of the actual side effects (unmaintained, buggy code) and provide other approaches than merely blacklisting.

Reply:
@rpannek and @jeremiah the answer to why are we doing this: because we have done it X number of time and we will probably do it a Y number of times. It will save someone some effort :). Is it good? Maybe not. Do we agree with it? Noup. Is it useful to have this documented? Definitively yes. That is at least my point here :).

Reply:
Okay, fair enough. But in that case I feel we should pretty much just reproduce the upstream documentation as faithfully as possible, i.e. https://wiki.yoctoproject.org/wiki/How_do_I#Q:_How_do_I_build_an_image_without_GPLv3_Licensed_packages_.3F

Reply:
I agree to what @breakreturn say. I would personally like to add a comment that this is not an endorsement or "disendorsement" of either habit on our side though.

Reply:
I agree to what @breakreturn say. I would personally like to add a comment that this is not an endorsement or "disendorsement" of either habit on our side though.